packages/suite/src/views/guide/GuideDefault.tsx

```diff
@@ -80,6 +80,7 @@ const ArticleLabel = styled.div`
     color: ${props => props.theme.TYPE_DARK_GREY};
 `;
 
+// TODO: if something has to be fetched on render of this component, make sure that it is fetched after guide is opened by a user
```

Comment:
Let's discuss whether to ban TODOs in code comments, because they are unmanageable. I would suggest to create an issue instead.

Reply:
I will address this comment immediately in my new branch